tests/libs/util/service_helper.cpp

```diff
@@ -35,21 +36,21 @@ service_install_helper::initialize()
             return error;
         }
 
-        // Install the service
+        // Install the service as LocalService.
```

Comment:
Why not use the install method in the ebpfsvc?

Reply:
We can use that only for the installation part, but this helper does more.

1. It deletes the service on exit.
2. If the service is already installed, it does not uninstall when the test exits. This is to ensure that running this test on the setup does not delete the service if someone has it already installed.
3. Starts / stops service and waits for it to reach the desired state before going to next step.